### PR TITLE
FMFR-1323-allow-update-of-the-account-status

### PR DESCRIPTION
**Pull request by domducsutcliffe**
https://crowncommercialservice.atlassian.net/jira/software/c/projects/FMFR/boards/71?modal=detail&selectedIssue=FMFR-1323

### CODE REVIEW

File: app/views/crown_marketplace/manage_users/show.html.erb [hunk 1]
```diff
@@ -40,9 +40,9 @@
         <dd class="govuk-summary-list__value">
           <%= govuk_tag_with_text(*enabled_disabled_status_tag(@user.account_status)) %>
         </dd>
-        <% if can_edit_user? %>
+        <% if can_edit_user? && permitted_sections.include?(:account_status) %>
```
Comment (tim-s-ccs) [inline app/views/crown_marketplace/manage_users/show.html.erb:43]: Why have you added this? Is there an admin user type who cannot edit the account status?

**Review by tim-s-ccs — CHANGES_REQUESTED** [review]
One small change and then I will approve

File: app/views/crown_marketplace/manage_users/show.html.erb (hunk 1 above)
Comment (domducsutcliffe) [reply app/views/crown_marketplace/manage_users/show.html.erb:43]: true no good reason

File: app/views/crown_marketplace/manage_users/show.html.erb [hunk 2]
```diff
@@ -40,9 +40,9 @@
         <dd class="govuk-summary-list__value">
           <%= govuk_tag_with_text(*enabled_disabled_status_tag(@user.account_status)) %>
         </dd>
-        <% if can_edit_user? %>
+        <% if can_edit_user%>
```
Comment (tim-s-ccs) [inline app/views/crown_marketplace/manage_users/show.html.erb:43]: You've taken too much off

```suggestion
        <% if can_edit_user? %>
```

**Review by tim-s-ccs — CHANGES_REQUESTED** [review]

File: app/views/crown_marketplace/manage_users/show.html.erb (hunk 2 above)
Comment (domducsutcliffe) [reply app/views/crown_marketplace/manage_users/show.html.erb:43]: di fixed

**Review by tim-s-ccs — APPROVED** [review]

**Pull request merged**